parquet-key-management/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "parquet-key-management"
+description = "Implements the Parquet Key Management Tools API in Rust to enable integration with a Key Management Server when using Parquet modular encryption."
+keywords = ["parquet", "encryption", "kms", "arrow"]
+version = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+rust-version = { workspace = true }
+
+[dependencies]
+base64 = { version = "0.22", default-features = false, features = ["std"] }
+parquet = { version = ">=55.0.0", default-features = false, features = ["encryption"] }
+ring = { version = "0.17", default-features = false, features = ["std"] }
+serde = { version = "1.0", default-features = false, features = ["derive"] }
+serde_json = { version = "1.0", default-features = false, features = ["std"] }
+
+[dev-dependencies]
+arrow = { version = ">=55.0.0", default-features = false }
+arrow-array = { version = ">=55.0.0", default-features = false }
```

Comment:
Maybe add arrow and arrow-array to the workspace to make sure that this and generate-test-files are using exactly the same version?

Reply:
Good idea, I've done that now thanks